Client/Ecosia/MMP/MMP.swift

```diff
@@ -23,7 +23,6 @@ struct MMP {
                                                   deviceManufacturer: DeviceInfo.manufacturer,
                                                   deviceModel: DeviceInfo.deviceModelName,
                                                   locale: DeviceInfo.currentLocale,
-                                                  country: DeviceInfo.currentCountry,
```

Comment:
Hmm.. I am wondering why this is in the changes 🤔 @lucaschifino @d4r1091 Can you tell whether this should be removed in `main`?

Reply:
@kimar can you tell why that change is there?

Reply:
> Hmm.. I am wondering why this is in the changes 🤔 @lucaschifino @d4r1091 Can you tell whether this should be removed in `main`?

any thoughts @d4r1091 @lucaschifino ?